lib/rucio/daemons/bb8/nuclei_background_rebalance.py

```diff
@@ -27,14 +27,38 @@
 min_total = 50 * 1E12
 total_rebalance_volume = 0
 
+
+# groupdisks
+def group_space(site):
+    """
+    groupdisks of given site
+    contributing to primaries
+    """
+    site_groupdisks = []
+    group_total = 0
+    try:
+        site_groupdisks = parse_expression('site=%s&type=GROUPDISK' % site)
```

Comment:
Nope  :
```
rucio list-rses --expression "site=BNL-ATLAS&spacetoken=ATLASDATADISK"
BNL-OSG2_DATADISK
```
vs :
```
rucio list-rses --expression "site=BNL-ATLAS&type=GROUPDISK"
BNL-OSG2_DET-SLHC
BNL-OSG2_PHYS-HI
```
```
rucio list-rses --expression "site=BNL-ATLAS&spacetoken=ATLASGROUPDISK"
BNL-OSG2_DET-SLHC
BNL-OSG2_PHYS-HI
```

Reply:
ok, but according to your example, there's no difference between spacetoken and type.

Reply:
Yes, but if you compare to :
```
rucio list-rses --expression "site=FZK-LCG2&spacetoken=ATLASDATADISK"
FZK-LCG2_PERF-EGAMMA
FZK-LCG2_DATADISK
FZK-LCG2_PERF-TAU                                                                                                                                                                                                                                                              
FZK-LCG2_PERF-IDTRACKING 
```
vs :

``` 
rucio list-rses --expression "site=FZK-LCG2&type=GROUPDISK"
FZK-LCG2_PERF-EGAMMA                                                                                                                                                                                                                                                           
FZK-LCG2_PERF-TAU                                                                                                                                                                                                                                                              
FZK-LCG2_PERF-IDTRACKING       
```

Reply:
this was done  Ithink